<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Daniel L. Contreras Pino</author>
 <class>GEARC_MainWindow</class>
 <widget class="QMainWindow" name="GEARC_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>152</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">GEARC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos/recursos.qrc">
    <normaloff>:/gearc.png</normaloff>:/gearc.png</iconset>
  </property>
  <widget class="QWidget" name="widget_central">
   <layout class="QGridLayout" name="layout_central"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Gearc</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actMenuSalir"/>
   </widget>
   <widget class="QMenu" name="menu_Ayuda">
    <property name="title">
     <string>&amp;Ayuda</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actMenuAbout"/>
    <addaction name="actMenuAboutQt"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menu_Ayuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actMenuSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actMenuAbout">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actMenuAboutQt">
   <property name="text">
    <string>Acerca de QT...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos/recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actMenuSalir</sender>
   <signal>triggered()</signal>
   <receiver>GEARC_MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
